### ISSUE TITLE
Ravencoin wallet 4.3.1.0 osx installation problem

### ISSUE THREAD

**Issue by bsdramer** [RavenProject/Ravencoin#841]
This issue tracker is only for technical issues related to bitcoin-core.

General ravencoin questions and/or support requests and are best directed to the [Ravencoin Discord](https://discord.gg/GwtXdyc).

For reporting security issues, please direct message one of the core developers in discord.

#### If so, please list the steps to reproduce below:
1. Download wallet release raven-4.3.1.0-os.dmg from github to macbook pro (2019) running Catalina v 10.15.7
2. drag raven icon to Applications folder
3. open Applications folder and double click on raven icon to begin the installation

### Expected behaviour
Tell us what should happen There should be a popup that begins the installation process. 

### Actual behaviour
Tell us what happens instead Nothing happens. A raven icon has appeared in the dock, but double clicking it just makes the icon bounce for a minute while nothing else happens. 

### Screenshots.
If the issue is related to the GUI, screenshots can be added to this issue via drag & drop.

### hat version of bitcoin-core are you using? I don't know what this means.
List the version number/commit ID, and if it is an official binary, self compiled or a distribution package such as PPA.W

### Machine specs:
- OS: Catalina v 10.15.7
- CPU: 2.6 GHz 6-Core Intel Core i7
- RAM: 16 GB 2667 MHz DDR4
- Disk size: I don't know where to look. It's a 2019 macbook pro.
- Disk Type (HD/SDD):

### Any extra information    I just don't understand why the installation of the wallet won't begin, and I have no idea what to do to get it going.

I clicked "view contents" to see what was in the downloaded dmg file, and I'm uploading a screenshot of those contents. I clicked on Raven-Qt and Terminal opened. I have no idea what to do with that. I'm completely lost. I just want to install my Raven wallet! 
<img width="1728" alt="Screen Shot 2020-10-26 at 1 08 13 PM" src="https://user-images.githubusercontent.com/73489982/97223430-fd738a00-178c-11eb-9f81-4dffa40762c2.png">

**Comment by Bidon15** [comment]
Hi @bsdramer ,

I've had no issues with installing v4.3.1 and latest(v.4.3.2.1) with the nearly the same setup that you have provided (Macbook 2018 + Catalina 10.15.6) 

I could suggest you to either check the "Security and Privacy" section of the "System Preferences" and find the section "Allow apps downloaded from" as this is where the Raven-Core Wallet App could be blocked from opening.

Just retry with the steps you have done and after clicking the app icon, navigate to the System Preferences to allow it to open.

Hope this helps,
Cheers

**Comment by bsdramer** [comment]
It turned out to be something else. The problem was not Raven-related at
all. There was something else in my system causing havoc with installing
ANYTHING. I found the solution by googling and googling until I found it.
I'm now going to copy and paste the notes I made:

WHEN AN APP WON’T INSTALL



How you know:  You downloaded the .dmg file, clicked on it, then you
dragged the icon of the new app into the Applications folder. The new icon
appears in the Applications folder. You double click the icon. In the Dock,
you see the icon is bouncing. After a minute, nothing else has happened and
the icon stops bouncing.



Steps to Take:

1.     Open the computer in Safe Mode:  Hit the Restart command. Hold down
the SHIFT key until the login screen appears. Let go. Enter password. The
computer is now operating in Safe Mode.

2.     Double click on the icon in the Applications folder and it should
open right up and begin installing itself.

3.     Restart the computer normally.


Maybe these notes will help someone else someday. Thanks for responding.
